tests/unit/cli/vyper_json/test_compile_json.py

```diff
@@ -318,4 +318,6 @@ def test_compile_json_with_abi_top(make_input_bundle):
 from . import stream
     """
     input_bundle = make_input_bundle({"stream.json": stream, "code.vy": code})
-    vyper.compiler.compile_code(code, input_bundle=input_bundle)
+    vyper.compiler.compile_code(
+        code, resolved_path=input_bundle.search_paths[0] / "code.vy", input_bundle=input_bundle
+    )
```

Comment:
why is this change needed?

Reply:
Because it imports by a relative path, it needs the current path of the file. By default it is left to unknown (`UNKNOWN_CONTRACT_NAME`), so then when we ask for the current directory (by `.parent) it gets evaluated to "." instead of the temporary directory. But i think you are right that i added it somewhere unnecessarily.

Reply:
here it's preferred to use FileInput which already has the path/resolved_path metadata on it -- i've pushed it here 3359eaed90c89b52a525f68888be4ad14644630e